app/utils/commands.js

```diff
@@ -0,0 +1,249 @@
+const { UntilMorning } = require('./untilMorning')
+const log = require('electron-log')
+
+const allOptions = {
+  title: {
+    long: '--title',
+    short: '-T',
+    description: 'Specify title for next break (Long or Mini)',
+    withValue: true
+  },
+  text: {
+    long: '--text',
+    short: '-t',
+    description: 'Specify text for next break (Long Break only)',
+    withValue: true
+  },
+  noskip: {
+    long: '--noskip',
+    short: '-n',
+    description: 'Do not skip directly to this break (Long or Mini)',
+    withValue: false
+  },
+  duration: {
+    long: '--duration',
+    short: '-d',
+    description: 'Specify duration for pausing breaks (Pause only) [infinitely|until-morning|seconds (as number)]',
```

Comment:
‘infinitely’ here should probably be ’indefinitely’, since that's what used later in the code and in the app's menu.

Reply:
Yeah sure, just a typo.